src/controllers/requests/singleUserRequest.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../../database/db');
+require('dotenv').config();
+
+
+const getSingleRequestByUser = async (request, response) => {
+    try{
+
+        const {id} = request.params;
+        const getQuery = 'SELECT * FROM requests WHERE id = $1';
```

Comment:
You would need to also query with the user's id, so as to ensure that the user is not trying to access someone else's request

Reply:
You should also add the check to send an error message if the specific request a user is trying to retrieve doesn't exist (after the query)